src/components/sidebar/sidebar.component.jsx

```diff
@@ -1,9 +1,13 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
 import React from "react";
 
-const Sidebar = (props) => {
+const Sidebar = () => {
   return (
     <div className="sidebar">
-      {props.children}
+      <a href="#">Mission</a>
```

Comment:
This file and `home.page.jsx` is confusing me, are we going to have the sidebar or not?